test/parallel/test-stream-end-of-streams.js

```diff
@@ -1,20 +0,0 @@
-'use strict';
-require('../common');
-const assert = require('assert');
-
-const { Duplex, finished } = require('stream');
-
-assert.throws(
-  () => {
-    // Passing empty object to mock invalid stream
-    // should throw error
-    finished({}, () => {});
-  },
-  { code: 'ERR_INVALID_ARG_TYPE' }
```

Comment:
This test shouldn't be deleted. I've shared some suggestions that should help you bring this back.

Reply:
restored